shell/pages/c/_cluster/apps/charts/install.vue

```diff
@@ -1029,6 +1030,19 @@ export default {
         setIfNotSet(cattle, 'windows.enabled', true);
       }
 
+      // Global prometheus storage values get added to the values
```

Comment:
This doesn't look like the right place to perform this cleanup; everything modified in `addGlobalValuesTo` is in `<rancher-monitroing chart>.values.global` while the prometheus configuration is within `<rancher-monitoring chart>.values.prometheus`...this should probably be addressed in a monitoring-scoped file instead of this general one